README.md

```diff
@@ -36,6 +38,11 @@ See related projects:
 See the [contribution guide](https://lsdb.readthedocs.io/en/stable/developer/contributing.html)
 for complete installation instructions and contribution best practices.
 
+## Citation
+
+If you use LSDB in your work, we ask the you cite the proceedings 
```

Comment:
I would suggest just small extension ``conference proceedings''. I was also thinking maaaybe ``ADASS conference proceedings'', or full name ``Astronomical Data...'', but I think conference proceedings might be the best.